tests/e2e/utils/lambda_layer/powertools_layer.py

```diff
@@ -31,6 +38,11 @@ def build(self) -> str:
 
         return str(self.output_dir)
 
+    def after_build(self):
+        subprocess.run(self.cleanup_command, shell=True)
```

Comment:
future note (not this PR): we should look to make it work without `shell=True`. Typically all you need is a list of commands, but I had issues with Layer build last time.